packages/theme-a11y/theme-a11y.test.js

```diff
@@ -4,65 +4,57 @@
 
 import path from 'path';
 import {
-  pageLinkFocus,
+  forceFocus,
   focusHash,
   bindInPageLinks,
   focusable,
   trapFocus,
   removeTrapFocus
 } from './theme-a11y';
 
-describe('pageLinkFocus()', () => {
+describe('forceFocus()', () => {
   beforeEach(() => {
     document.body.innerHTML =
       '<div id="nonFocusableElement"></div>' +
       '<button id="focusableElement" tabIndex=3 class="myClass">';
   });
 
   test('is a function exported by theme-a11y.js', () => {
-    expect(typeof pageLinkFocus).toBe('function');
+    expect(typeof forceFocus).toBe('function');
   });
 
   test("adds a 'tabindex=-1' attribute", () => {
     const nonFocusableElement = document.getElementById('nonFocusableElement');
     const focusableElement = document.getElementById('focusableElement');
 
-    pageLinkFocus(nonFocusableElement);
+    forceFocus(nonFocusableElement);
     expect(nonFocusableElement.tabIndex).toBe(-1);
 
-    pageLinkFocus(focusableElement);
+    forceFocus(focusableElement);
     expect(focusableElement.tabIndex).toBe(-1);
   });
 
   test('reserves the original tabindex value in a data-tabIndex attribute', () => {
     const focusableElement = document.getElementById('focusableElement');
 
     focusableElement.tabIndex = 3;
-    pageLinkFocus(focusableElement);
+    forceFocus(focusableElement);
 
     expect(parseInt(focusableElement.dataset.tabIndex)).toBe(3);
   });
 
   test('adds a focus state', () => {
     const focusableElement = document.getElementById('focusableElement');
 
-    pageLinkFocus(focusableElement);
+    forceFocus(focusableElement);
 
     expect(document.activeElement).toBe(focusableElement);
   });
 
-  test("adds the default class name 'js-focus-hidden'", () => {
-    const focusableElement = document.getElementById('focusableElement');
-    const defaultClass = 'js-focus-hidden';
-    pageLinkFocus(focusableElement);
-
-    expect(focusableElement.classList.contains(defaultClass)).toBeTruthy();
-  });
-
   test('adds an overrided class name', () => {
```

Comment:
`adds an overrided class name` -> `adds a class name if specified in the options`